src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1613,29 +1615,11 @@ private void InitializeIncludedNodes()
             };
 
             SearchModel?.Add(symbolSearchElement);
-            SearchModel?.Add(outputSearchElement);
-
-            //Adding this nodes are breaking the tests (due that we have two input and two output nodes):
-            //WhenHomeWorkspaceIsFocusedInputAndOutputNodesAreMissingFromSearch
-            //WhenStartingDynamoInputAndOutputNodesAreNolongerMissingFromSearch
-            // New index process from Lucene, adding missing nodes: Code Block, Input and Output
-            var ele = AddNodeTypeToSearch(outputData);      
-            if (ele != null)
-            {
-                AddNodeTypeToSearchIndex(ele, iDoc);
-            }
+            AddNodeTypeToSearchIndex(symbolSearchElement, iDoc);
```

Comment:
@RobertGlobant20 Didn't we add the check intentionally? Should we merge this only after we have the corresponding unit test added?

Reply:
If you are refering to the check:
`if (ele != null)`
I think there is no need to check it due that we are not using the AddNodeTypeToSearch(outputData); method  anymore (that could return null) and if you see the code some lines above you will see that those instances are created (see screenshot below).

The unit test only validates that the Input, Output and Code Block nodes are found with the Lucene Search results then it won't catch if they are duplicated visually in Library (As far as I remember Lucene Search doesn't return duplicated items), then I think this change can be merged

![image](https://github.com/DynamoDS/Dynamo/assets/61755417/55b6ea80-e5d8-4a99-9414-38d12e373269)